packages/myst-cli/src/transforms/crossReferences.ts

```diff
@@ -74,12 +79,93 @@ export async function fetchMystLinkData(session: ISession, node: Link, vfile: VF
   return fetchMystData(session, node.dataUrl, node.urlSource, vfile);
 }
 
+const MYST_SPEC_VERSION = '0';
+
+function upgradeMystData(version: string, data: any): MystData {
+  return data;
+}
+
+async function stepwiseDowngrade(
+  session: ISession,
+  fromVersion: string,
+  toVersion: string,
+  vfile: VFile,
+  data: any,
+): Promise<any> {
+  const downgradeCachePath = `myst-downgrade-${fromVersion}-${toVersion}.mjs`;
+  if (
+    !checkCache(session, downgradeCachePath, {
+      maxAge: 7, // days
+    })
+  ) {
+    try {
+      const response = await fetch(`http://localhost:9000/${downgradeCachePath}`);
```

Comment:
What are your thoughts on just treating this as a normal npm package that we install in the `_build` directory and check for updates (in the same way we do for mystmd now)?

Then there would be a version check against npm for that package, an npm install (already doing that for the themes).

That would mean we don't have to host these, we have versions, publishing, deprecations, etc. AND other tools can use this in a normal way - it is just a package.